### ISSUE TITLE
Segfault at launch on Ubuntu 20.04

### ISSUE THREAD

**Issue by leogama** [ito32bit/qterminal-snap-packaging#1]
**Describe the bug**

Hi, I've tried to launch QTerminal - snap on Ubuntu 20.04 but got a segmentation fault. No window, no clue, just a hard crash...

**To Reproduce**

1. Install the snap in classic mode, as required
2. Launch the app from app menu -> nothing happens
3. Launch the app from command-line with `snap run qterminal-snap` -> segmentation fault

The error messages:
```
Qt: Session management error: None of the authentication protocols specified are supported
QDir::exists: Empty or null file name
QDir::exists: Empty or null file name
QDir::exists: Empty or null file name
zsh: segmentation fault (core dumped)  snap run qterminal-snap
```

**Expected behavior**

The app launches? A more helpful error message and a clean exit at least.

**Snap information:**

- Version: 210210.amd64-796.339
- Rev: 480
- Tracking: latest/edge

**System infomation:**

- snap: 2.48.3+20.04
- snapd: 2.48.3+20.04
- series: 16
- distribution: 20.04
- kernel: 5.4.0-65-generic
- desktop: Gnome (either on X or on Wayland)

**Additional context**
I'm trying to track down a bug I have with QTerminal after the upgrade from 18.04. I hoped I could use this snap with a more recent version to check if the bug is already solved (and maybe keep using it!).

If I can help with anything else, just ask. Thank you for your time.

**Comment by ito32bit** [comment]
Thank you for your report.
Could you remove and install?

```sh
sudo snap remove --purge qterminal-snap
sudo snap install --classic --edge qterminal-snap
```

**Comment by leogama** [comment]
It was a fresh install of the snap already, but I did what you suggested anyway. I'm getting the same error. `qterminal --version` and `qterminal --help` works, though.

Here are a `strace` output and an AppArmor crash report (of a normal run, I think) that I manage to get:
- [qterminal-snap.strace.gz](https://github.com/ito32bit/qterminal-snap-packaging/files/5977861/qterminal-snap.strace.gz)
- [qterminal-snap.crash.gz](https://github.com/ito32bit/qterminal-snap-packaging/files/5977860/qterminal-snap.crash.gz)

Hope it helps. :slightly_smiling_face:

**Comment by ito32bit** [comment]
What is your $SHELL? If $SHELL contains "zsh", try following command:

```sh
SHELL=/bin/bash snap run qterminal-snap
```

**Comment by leogama** [comment]
Yes, I use zsh. Tried this command, but got an identical crash. I've tried even with a "guest" user account in which I didn't make any settings changes for no luck. Sorry.
